index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Linkbrary</title>
+    <meta name="title" content="Linkbrary" />
+    <meta name="description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://linkbrary.codeit/" />
+    <meta property="og:title" content="Linkbrary" />
+    <meta property="og:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <meta property="og:image" content="https://metatags.io/images/meta-tags.png" />
+
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content="https://linkbrary.codeit/" />
+    <meta property="twitter:title" content="Linkbrary" />
+    <meta property="twitter:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <meta property="twitter:image" content="https://metatags.io/images/meta-tags.png" />
+    <link rel="stylesheet" href="styles/index_style.css" />
+    <link rel="stylesheet" href="styles/reset.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="nav_fixed">
+        <a href="index.html">
+          <img src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
+        </a>
+        <a href="signin.html">
+          <span class="signin">로그인</span>
+        </a>
+        </div>
+      </nav>
+      <div class="title_header">
+      <div class="title">
+        <h1 class="slogan">
+          <span class="slogan_gradient background-clip-text">세상의 모든 정보</span>를<span class="space"></span>
```

Comment:
빈 block box를 하나 추가하기보다는 의미있는 block으로 감싸는 게 어떨까 싶습니다!
```suggestion
          <p>
            <span class="slogan_gradient background-clip-text">세상의 모든 정보</span>를
          </p>
```